15/umbraco-cms/extending/packages/creating-a-package.md

```diff
@@ -71,75 +74,100 @@ The files that we created from the [Creating a Custom Dashboard Tutorial](../../
+1. Install the latest [.NET SDK](https://dotnet.microsoft.com/download).
+2. Run `dotnet new install Umbraco.Templates` to install the project templates.
+3. Run `dotnet new umbraco-extension -n CustomWelcomeDashboard` to create a new package project.
 
-```
-dotnet new umbracopackage --name CustomWelcomeDashboard
-```
+This command will generate the following files:
 
-{% hint style="info" %}
-For a guide on how to install the project templates, follow the 2 steps listed in the [Install the template section](../../fundamentals/setup/install/install-umbraco-with-templates.md).
-{% endhint %}
+![Content of an empty package](images/empty-package-from-template-v15.png)
 
-The outcome is the files generated below:
+Apart from the project file, the generated structure includes key folders and files that are essential for building an Umbraco extension.
 
-![Content of an empty package](<../../../../10/umbraco-cms/extending/packages/images/empty-package-from-template (1).png>)
+- **Client** - where you can place any front-end assets, such as JavaScript, CSS, and Angular views, that will be used in the backoffice.
+- **Composers** - intended for C# composer classes, which are used to register dependencies and configure services in Umbraco.
+- **Controllers** - where you can add API controllers if your package requires custom endpoints.
+- **Constants.cs** - serves as a placeholder for any global constants your package may need.
 
-Apart from the project file, you can find an empty `package.manifest` inside the **App\_Plugins** folder, which we will replace with the one created from the [Creating a Custom Dashboard Tutorial](../../tutorials/creating-a-custom-dashboard/). But more importantly, it also contains a `build/CustomWelcomeDashboard.targets` file.
+Unlike previous versions, the `umbraco-extension` template does not generate a `package.manifest` file or an `App_Plugins` folder by default. If your package includes client-side assets for the backoffice, you will need to manually create an `App_Plugins/{YourPackageName}` folder.
 
-This file contains an `msbuild` target that is executed when a project has a dependency on this package. It copies the `App_Plugins` folder into the project on the build. This is required for having Umbraco packages in a NuGet package format.
+Additionally, the `.csproj` file is configured to support NuGet packaging, allowing you to distribute your extension. If you plan to include custom C# logic, the files you place in the root folder will be compiled into the package DLL.
 
-{% hint style="info" %}
-If you are planning to overwrite the contents of the **App\_Plugins** folder, make sure that the subfolder containing your package contents has the same name as the one you specified after the `--name` flag and that the `package.manifest` has the correct path references to your files.
-{% endhint %}
+#### Post-Installation Build Script
```

Comment:
I think this (and the section below it with the workaround) should be added to the "generate an empty package using a template" steps, right?
It is something that happens when you run the third step, so to me it would make sense if this information was added there.

Or perhaps it could just be put into a note? 🤔

Reply:
I have added it in a Note below step 3..